forms-flow-web/src/helper/helper.js

```diff
@@ -54,11 +54,13 @@ const getEmployeeNameFromSubmission = (form, submission) => {
   } else if (
     formNameLower.includes(
       FORM_NAMES.INFLUENZA_WORKSITE_REGISTRATION.toLowerCase()
+    ) || formNameLower.includes(
+      FORM_NAMES.INFLUENZA_CLINIC_WORKSITE_REGISTRATION.toLowerCase()
     )
   ) {
     const firstName = formData?.primary_first_name || "";
     const lastName = formData?.primary_last_name || "";
-    submitterName = `${firstName} ${lastName}`;
+    submitterName = `${firstName} ${lastName}`.replace(/''/g, "'");
```

Comment:
Wondering why we need this Regex

Reply:
For Cold/Flu we are sending '' to ODS and display ' in our system. Here we receive row data that saved and that contain '' so converting that to ' (single quote). It can be remove in future once we have fix for the same in ODS system.

Reply:
Sounds good. Can you add a todo to make sure we can find it later? Thanks